spec/controllers/admin/support_notices_controller_spec.rb

```diff
@@ -0,0 +1,175 @@
+require "spec_helper"
+
+describe Admin::SupportNoticesController do
+  include LoginMacros
+  include RedirectExpectationHelper
+
+  let(:admin) { create(:admin, roles: ["superadmin"]) }
+  let(:support_notice) { create(:support_notice) }
+  let(:support_notice_params) { attributes_for(:support_notice, notice: "New notice content") }
+
+  shared_examples "only authorized admins are allowed" do |authorized_roles:|
```

Comment:
We have a `an action only authorized admins can access` shared example (for all specs) that should already cover most of this. I think it would be better to use that and make a separate example for what it doesn't cover